test/fixtures/user_profiles.yml

```
@@ -0,0 +1,38 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined. If you add columns to the
+# model remove the '{}' from the fixture names and add the columns immediately
+# below each fixture, per the syntax in the comments below
+#
+one:
+  user_id: 1
+  name: Taro Yamada
+  name_kana: ヤマダ タロウ
+  nickname: ヤタロー
+  email: taro@example.com
+  major_subject: 情報アーキテクチャコース
```

デザイン上では

```suggestion
  major_subject: 情報アーキテクチャ
```

にしてるので、もし合わせるのであればコースはトリでいいかもです。

---

132b6a8c11098dcd18d8c3b9d54af38dad209dba
こちらで対応しました。